File: app/src/main/java/com/shafa/ali/kavir_msg/models/RegisterModel.java
package com.shafa.ali.kavir_msg.models;

import com.google.gson.annotations.SerializedName;

public class RegisterModel {
    @SerializedName("result")
    private String status;

    @SerializedName("user_id")
    private String userId;

    @SerializedName("message")
    private String message;

    @SerializedName("s")
    private String scode;

    @SerializedName("mid")
    private String mid;

    public RegisterModel() {
    }

    public String getStatus() {
        return status;
    }

    public void setStatus(String status) {
        this.status = status;
    }

    public String getUserId() {
        return userId;
    }

    public void setUserId(String userId) {
        this.userId = userId;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }

    public String getScode() {
        return scode;
    }

    public void setScode(String scode) {
        this.scode = scode;
    }

    public String getMid() {
        return mid;
    }

    public void setMid(String mid) {
        this.mid = mid;
    }
}

File: app/src/main/java/com/kavirelectronic/ali/kavir_info/models/RegisterModel.kt
package com.kavirelectronic.ali.kavir_info.models

import com.google.gson.annotations.SerializedName

class RegisterModel {
    @SerializedName("result")
    var status: String? = null

    @SerializedName("user_id")
    var userId: String? = null

    @SerializedName("message")
    var message: String? = null

    @SerializedName("s")
    var scode: String? = null

    @SerializedName("mid")
    var mid: String? = null

}